pyfolio/utils.py

```diff
@@ -108,29 +96,40 @@ def default_returns_func(symbol):
         Daily returns for the symbol.
          - See full explanation in tears.create_full_tear_sheet (returns).
     """
+    if start is None:
+        start = '1/1/1970'
+    if end is None:
+        end = datetime.now()
 
-    rets = None
-    if symbol == 'SPY':
-        filepath = data_path('spy.h5')
-        try:
-            # If it's been less than a day since we got benchmark
-            if datetime.now() - pd.to_datetime(
-                    getmtime(filepath), unit='s') < pd.Timedelta(days=1):
-                rets = pd.read_hdf(filepath, 'df')
-        except:
-            pass
+    start = pd.Timestamp(start)
+    end = pd.Timestamp(end)
 
-    if rets is None:
-        px = web.get_data_yahoo(symbol, start='1/1/1970')
+    def get_symbol_from_yahoo(symbol, start=None, end=None):
```

Comment:
Is this nested because we don't wan't to expose the function? It seems pretty harmless and could be useful for people if we expose it.